cypress/e2e/dfeAdmin/manageServices/editService-dfeAdmin-laAndOrgDetails.spec.js

```diff
@@ -0,0 +1,145 @@
+
+        // And I am on the view services page
+        cy.navigateToDfeAdminViewServicesPage()
+    })
+
+    it('can edit associated LA for a LA service', () => {
+                // When I apply a filter
+        cy.get('[id="service-name"]').type('Edit LA Service');
+        cy.get('[type="submit"]').contains('Apply filter').click();
+
+        // And I click to view a service   
+        cy.get('.govuk-table__cell a')
+        .eq(0)
+        .click();
+
+        // When I click the change link for Local authority
+        cy.clickChangeLink('Local authority');
+
+        // And I change the Local authority to a different one
+        cy.get('[id="select__option--0"]').invoke('text').then(($value) => {
+            if ($value === 'Tower Hamlets Council') {
+                cy.get('[id="select"]').clear();
+                cy.get('[id="select"]').type('Bristol County');
+            }
+            else {
+                cy.get('[id="select"]').clear();
+                cy.get('[id="select"]').type('Tower Hamlets');
+            }
+        });
+    
+        cy.get('[id="select__option--0"]').click();
+
+        // And I click continue
+        cy.get('#main-content > div > div > form > button').click();
+
+        // Then view service page is displayed
+        cy.checkPageUrlContains(expectedEditServicePageUrl);
+
+        // And I click save
+        cy.get('#main-content > div > div > form > button').click();
+
+        // Then the changes saved page is displayed
+        cy.checkPageUrlContains(expectedConfirmationPageUrl); 
+    })
+
+    it('can edit associated LA for a VCS service', () => {
+        // When I apply a filter
+        cy.get('[id="service-name"]').type('Edit VCS Service');
+        cy.get('[type="submit"]').contains('Apply filter').click();
+
+        // And I click to view a service   
+        cy.get('.govuk-table__cell a')
+        .eq(0)
+        .click();
+
+        // When I click the change link for Local authority
+        cy.clickChangeLink('Local authority');
+
+        // And I change the Local authority to a different one
+        cy.get('[id="select__option--0"]').invoke('text').then(($value) => {
+            if ($value === 'Tower Hamlets Council') {
+                cy.get('[id="select"]').clear();
+                cy.get('[id="select"]').type('Bristol County');
+            }
+            else {
+                cy.get('[id="select"]').clear();
+                cy.get('[id="select"]').type('Tower Hamlets');
+            }
+        });
+
+        // And I click continue
+        cy.get('#main-content > div > div > form > button').click();
+
+        // Then which organisation page is displayed
+        cy.checkPageUrlContains(expectedOrganisationPageUrl); 
+
+        // When I change the organisation
+        cy.get('[id="select"]').clear();
+        cy.get('[id="select"]').type(getRandomLetter());
+
+        var vcsOption = "select__option--" + getRandomInt(0, 2).toString();
```

Comment:
Just a question on the use of selecting options with a random number, do we have a guarantee that there will always be 2 organisations each time the list selector is triggered? In the case there is only one organisation, but the random number is 1, it will throw an index error as the one org is at index 0

Reply:
Good question: We have a guarantee for Tower Hamlets and Bristol (which are the two LAs I've hardcoded as our test LA) - I've just double checked. And another thing to point out is getRandomLetter function does not return q or x as there are no results for those letters.